HOWTORELEASE

```diff
@@ -0,0 +1,54 @@
+HOWTO RELEASE THE PROJ-DATUMGRID PACKAGE
+========================================
+
+Prerequisites:
+
+    - PROJ 4.9.3 or newer. Explained here why:
+      http://lists.maptools.org/pipermail/proj/2016-October/007544.html
+    - CMake/CPack
+
+1. Adjust the version number:
+
+    If preparing a release candidate "RCx" should be added
+    to PROJ_DATUMGRID_VERSION_MINOR in CMakeLists.txt.
+
+    If preparing the final release any postfixes to the version
+    number should be stripped.
+
+2. Create tarball and zip:
+
+    mkdir build
+    cd build
+    cmake ..
+    make dist
+
+3. Check repo for grid changes:
+
+    git status
+
+4. Check contents of tarball and zip
+
+    tar tvf proj-datumgrid-1.7RC1.tar.gz
+    unzip -l proj-datumgrid-1.7RC1.zip
+
+5. Upload the files to download.osgeo.org/proj
+
+    If you are preparing a release, you know how to do this.
+
+6. Clean up
+
+    cd ..
+    rm -rf build
+
```

Comment:
If the git repository is clear, the release needs to be tagged.
```
git tag $(echo $(grep "set(PROJ_DATUMGRID_VERSION_MAJOR" ../CMakeLists.txt | awk '{print $2}' | sed 's/)//').$(grep "set(PROJ_DATUMGRID_VERSION_MINOR" ../CMakeLists.txt | awk '{print $2}' | sed 's/)//'))
git push --tags
```

After the tag is pushed a release should be created on GitHub.